proto/osmosis/gamm/pool-models/balancer/balancerPool.proto

```diff
@@ -79,12 +79,7 @@ message PoolParams {
     (gogoproto.moretags) = "yaml:\"swap_fee\"",
     (gogoproto.nullable) = false
   ];
-  string exit_fee = 2 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-    (gogoproto.moretags) = "yaml:\"exit_fee\"",
-    (gogoproto.nullable) = false
-  ];
-  SmoothWeightChangeParams smooth_weight_change_params = 3 [
```

Comment:
I think we should be able to make this field simply as `reserved`. That way, we would not need to generate v2 types and perform risky and complex migrations.